### PR TITLE
MM-37746 Fix premature truncation of names in emoji picker

### PR DESCRIPTION
**Pull request by Roy-Orbison**
#### Summary
Fix premature truncation of names in emoji picker. Generally simplify style now that aliases are shown on the same line as the primary name.

#### Ticket Link
https://github.com/mattermost/mattermost-server/issues/18430  
https://mattermost.atlassian.net/browse/MM-37746

#### Screenshots
|  before  |  after  |
|----|----|
| Screenshot_20211029_110846 | Screenshot_20211028_115539 |

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Comment by Willyfrog** [comment]
yes, make-emojis was the script used with the old library, while emoji_gen is the new one. We don't use the ruby ones (make-emojis) anymore so it could be deleted along with the rest of ruby files.

**Comment by Roy-Orbison** [comment]
Okay. So why do the rules in `_emojisprite.scss` have a different order to the CSS template literal in `emoji_gen.mjs`? Is there some other manual or automated editing of the generated file?

**Comment by Willyfrog** [comment]
seems like a bad merge reverted the changes partially: https://github.com/mattermost/mattermost-webapp/pull/8273/files#diff-ba941149d45ca4cceade5b5fdd1897ad34adb009deb046feb365a5556003f717

if you run `make emojis` it should fix it

**Comment by Roy-Orbison** [comment]
Perhaps they were reordered to appease CI, after generation, because CI can't check CSS rule order inside a JS file?

**Comment by emilyacook** [comment]
Thanks for participating in Hacktoberfest! You can claim your sticker set here: https://get.printfection.com/hacktober21/4144583267 @Roy-Orbison

File: build/emoji/emoji_gen.mjs [hunk 1]
```diff
@@ -337,38 +337,40 @@ for (const key of emojiFilePositions.keys()) {
...
-    -moz-transform: scale(0.55);
+    width: ${EMOJI_SIZE_PADDED}px;
+    max-width: none;
+    height: ${EMOJI_SIZE_PADDED}px;
     background-repeat: no-repeat;
     cursor: pointer;
-    height: ${EMOJI_SIZE_PADDED}px;
-    max-width: none;
-    width: ${EMOJI_SIZE_PADDED}px;
-    padding: 0 10px 0 0;
```
Comment (Willyfrog) [inline build/emoji/emoji_gen.mjs:349]: did we remove this one?

**Review by Willyfrog — APPROVED** [review]
LGTM added a comment because I'm not sure if that removal was intentional

File: build/emoji/emoji_gen.mjs (hunk 1 above)
Comment (Roy-Orbison) [reply build/emoji/emoji_gen.mjs:349]: The padding removal was intentional. I think it was compensation for some of the odd relative positioning.

**Review by hmhealey — APPROVED** [review]
Looks good to me as well then!

**Review by jgilliam17 — APPROVED** [review]
Thanks @Roy-Orbison 
Tested, looks good to merge.
- Verified emoji aliases are now truncated as expected.

**Pull request merged**